lib/api-client.js

```diff
@@ -6,22 +6,21 @@ var apiClient = new JSONAPIClient(config.host + '/api', {
   'Accept': 'application/vnd.api+json; version=1',
 }, {
   beforeEveryRequest: function() {
+    console.log('using local panoptes-client');
```

Comment:
will remove, keeping momentarily while try to stage branch of PFE with dependency as GitHub branch that itself has a dependency as GitHub branch.